packages/atclient/include/atclient/metadata.h

```diff
@@ -116,15 +117,15 @@ typedef struct atclient_atkey_metadata {
   // Represents the amount of time for atkey to exist from the point at birth to the point at death.
   // Example ttl=86400 means the atkey will live for a day.
   // This field is read from protocol string and and set by the developer.
-  // This field is written to protocol string by the SDK. (See atclient_atkey_metadata_to_protocolstr)
+  // This field is written to protocol string by the SDK. (See atclient_atkey_metadata_to_protocol_str)
```

Comment:
major change 2: protocolstr to protocol_str so that it's consistent with the convention of protocol_strlen